app/forms/nsm/steps/letters_calls_form.rb

```diff
@@ -39,13 +39,13 @@ def letters_row
         [
           translate(:letters),
           {
-            text: NumberTo.pounds(letters_before_uplift),
+            text: NumberTo.pounds((letters_before_uplift if letters.presence)),
```

Comment:
this might be me being dim and not seeing something obvious, but where is this letters method coming from?

Reply:
It's an attribute defined on line 7. `letters_before_uplift` used to be nil of `letters` hadn't been set, but now it will be `0`, so to preserve existing behaviour I'm checking for `letters` being set here.